frontend/src/component/insights/componentsChart/CreationArchiveChart/CreationArchiveChart.tsx

```diff
@@ -35,93 +65,240 @@ type RawWeekData = {
+                                            chart,
+                                        );
+                                    const filtered = original.filter(
+                                        (item) =>
+                                            !flagTypeNames.includes(
+                                                item.text || '',
+                                            ),
+                                    );
+
+                                    filtered.push({
+                                        text: 'Created Flags',
+                                        fillStyle: theme.palette.success.main,
+                                        strokeStyle: theme.palette.success.main,
+                                        lineWidth: 0,
+                                        hidden: false,
+                                        index: filtered.length,
+                                        datasetIndex: -1,
+                                    });
+
+                                    return filtered;
+                                },
+                            },
+                        },
+                        tooltip: {
+                            enabled: false,
+                            position: 'nearest',
+                            external: createTooltip(setTooltip),
+                        },
+                    },
+                    locale: locationSettings.locale,
+                    scales: {
+                        x: {
+                            type: 'time',
+                            time: {
+                                unit: 'week',
+                                tooltipFormat: 'PPP',
+                            },
+                        },
+                        y: {
+                            type: 'linear',
+                            position: 'left',
+                            beginAtZero: true,
+                            title: {
+                                display: true,
+                                text: 'Number of flags',
+                            },
+                        },
+                        y1: {
+                            type: 'linear',
+                            display: true,
+                            position: 'right',
+                            beginAtZero: true,
+                            title: {
+                                display: true,
+                                text: 'Ratio',
+                            },
+                            grid: {
+                                drawOnChartArea: false,
+                            },
+                            ticks: {
+                                callback: (value) => `${value}%`,
+                            },
+                        },
+                    },
+                }}
+                height={100}
+                width={250}
+            />
+            {tooltip?.dataPoints?.some(
```

Comment:
There are different tooltips for different datasets and they must be shown conditionally.